### PR TITLE
add camera mask check to visit

### PR DESCRIPTION
**Pull request by super626**
Others override visit should check camera mask by calling isVisitableByVisitingCamera.

use renderer passed instead of get from global for Sprite3D::draw function

### CODE REVIEW

File: cocos/2d/CCNode.h [hunk 1]
```diff
@@ -1521,6 +1521,9 @@ class CC_DLL Node : public Ref
     bool doEnumerate(std::string name, std::function<bool (Node *)> callback) const;
     bool doEnumerateRecursive(const Node* node, const std::string &name, std::function<bool (Node *)> callback) const;
     
+    //check whether this camera mask is visible by the current visiting camera
+    bool checkCameraMask() const;
```
Comment (minggo) [inline cocos/2d/CCNode.h]: How about rename to `isVisibleByVisitingCamera`?

Comment (super626) [inline cocos/2d/CCNode.h]: isVisibleByVisitingCamera seems that it is going to do frustum culling.

File: tools/cocos2d-console [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 5db5a4283aa22e362901e2924146ad66c2e1484f
+Subproject commit 4b020e4900e601a2d5845095f2e6970c13c59613
```
Comment (minggo) [inline tools/cocos2d-console]: Please not change it.

**Comment by super626** [comment]
Have changed

**Pull request merged**